watchman/checks.py

```diff
@@ -81,3 +95,7 @@ def email():
 
 def storage():
     return {"storage": _check_storage()}
+
+
+def amqp():
+    return {'amqp': _check_ampq(settings.AMQP_CONNECTIONS)}
```

Comment:
I am not sure about the composition of users of this package, but you had created https://github.com/mwarkentin/django-watchman/issues/21 to do this every thing and was implemented to a similar net effect in https://github.com/mwarkentin/django-watchman/issues/77#issuecomment-215202087